src/layouts/Sidebar.tsx

```diff
@@ -264,6 +297,16 @@ const NotificationMenu = memo(
         suffix: <JobLoggerStateIndicator state={jobLoggerState} />,
         onClick: onClickJobLoggerButton,
       },
+      {
+        iconName: Icons.SIGN_OUT,
+        children: t('menu.logout'),
+        onClick: async () => {
+          removeAuthenticationCookies();
+          await queryClient.invalidateQueries('user');
+
+          window.location.assign('/api/auth/logout');
```

Comment:
Would maybe use `router.push` here to be sure it doesn't do anything special underneath in addition to switching location

Reply:
True, I think it was like this for a reason but I don't remember and it seems to work with `router.push`